src/UI/UIView.cpp

```diff
@@ -1248,12 +1618,100 @@ void UIView::load_user_session(std::string filePath)
       {
         load_trans_functions(loadFile);
       }
+      else if (tag == "POI")
+      {
+        load_camera_poi(loadFile, std::stoi(vals[1]));
+      }
     }
     loadFile.close();
+    m_controller_app.get_trackball_camera().reset_camera();
+  }
+}
 
+void UIView::save_simulation_states(std::ofstream& saveFile, int num_camera_poi)
+{
+  if (saveFile.is_open())
+  {
+
+    auto sim_iterator = m_controller_app.get_simulation().get_simulation_states().begin();
+    for (sim_iterator; sim_iterator != m_controller_app.get_simulation().get_simulation_states().end(); sim_iterator++)
+    {
+      saveFile << sim_iterator->poi.label << " "
+        << std::to_string(sim_iterator->poi.eye.x) + " " + std::to_string(sim_iterator->poi.eye.y) + " " + std::to_string(sim_iterator->poi.eye.z) + " "
+        << std::to_string(sim_iterator->poi.target.x) + " " + std::to_string(sim_iterator->poi.target.y) + " " + std::to_string(sim_iterator->poi.target.z) + " "
```

Comment:
would it be easier/more maintainable to use a self-documenting file format such as JSON instead of a plaintext file with a set order to the values?

Reply:
I'll create an issue with this new feature.

Reply:
https://github.com/brown-ccv/VR-Volumeviewer/issues/47